src/Core/CustomRedirects/CustomRedirectCollection.cs

```diff
@@ -14,12 +17,45 @@ public class CustomRedirectCollection: CollectionBase
         /// <summary>
         /// Hashtable for quick lookup of old urls
         /// </summary>
-        private readonly Hashtable _quickLookupTable = new Hashtable(StringComparer.InvariantCultureIgnoreCase);
+        private readonly Dictionary<string, CustomRedirect> _quickLookupTable = new Dictionary<string, CustomRedirect>();
+
+        /*[CanBeNull]*/
```

Comment:
is this CodeContracts attribute?

Reply:
https://www.nuget.org/packages/JetBrains.Annotations/ - if you use resharper it will show possible NREs with these attributes

Reply:
@valdisiljuconoks , is it ok to install the JetBrains.Annotations package?